contracts/root/withdrawManager/WithdrawManager.sol

```diff
@@ -24,6 +24,11 @@ contract WithdrawManager is WithdrawManagerStorage /* , IWithdrawManager */ {
   using RLPReader for RLPReader.RLPItem;
   using Merkle for bytes32;
 
+  modifier isProofValidator() {
```

Comment:
cool! The only reason I thought it should move to registry is because it is being used at multiple places.

Reply:
em!